source/deploying-with-puppet/wazuh-puppet-module/reference-wazuh-puppet/wazuh-manager-class.rst

```diff
@@ -491,8 +491,8 @@ $vulnerability_detector_interval
 
   `Default 5m`
 
-$vulnerability_detector_ignore_time
-  Time during which vulnerabilities that have already been alerted will be ignored.
+$vulnerability_detector_min_full_scan_interval
+  Time interval in which a full scan will be triggered if the database of vulnerabilities is updated with new CVEs information.
```

Comment:
Maybe this sounds like if the full_scan will be triggered inside the time interval and not after.
It can be reworded like:
`Time interval after which a full scan will be triggered if the database of vulnerabilities is updated with new CVEs information.`

Reply:
Fixed!